src/plugins.js

```diff
@@ -0,0 +1,131 @@
+export default {
+  // es2015
+  "transform-es2015-arrow-functions": {
+    edge: 13,
+    firefox: 45,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-block-scoped-functions": {
+    edge: 11,
+    firefox: 46,
+    chrome: 41,
+    opera: 28,
+  },
+  "transform-es2015-block-scoping": {
+    edge: 11,
+    firefox: 36,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-classes": {
+    edge: 13,
+    firefox: 45,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-computed-properties": {
+    edge: 12,
+    firefox: 34,
+    chrome: 44,
+    opera: 31
+  },
+  "check-es2015-constants": {
+    edge: 11,
+    firefox: 36,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-destructuring": {
+    edge: 14,
+    firefox: Number.NEGATIVE_INFINITY,
+    chrome: 52,
+    opera: 39,
+  },
+  "transform-es2015-for-of": {
+    edge: Number.NEGATIVE_INFINITY,
+    firefox: Number.NEGATIVE_INFINITY,
+    chrome: 51,
+    opera: 38,
+  },
+  "transform-es2015-function-name": {
+    edge: Number.NEGATIVE_INFINITY,
+    firefox: Number.NEGATIVE_INFINITY,
+    chrome: 52,
+    opera: 39,
+  },
+  "transform-es2015-literals": {
+    edge: 12,
+    firefox: 36,
+    chrome: 41,
+    opera: 28
+  },
+  "transform-es2015-object-super": {
+    edge: 13,
+    firefox: 45,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-parameters": {
+    edge: 14,
+    firefox: Number.NEGATIVE_INFINITY,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-shorthand-properties": {
+    edge: 12,
+    firefox: 33,
+    chrome: 43,
+    opera: 30
+  },
+  "transform-es2015-spread": {
```

Comment:
Both this and destructuring are needed if core-js needs to polyfill `Symbol` (array rest/spread actually uses iterators). This is probably not a problem in practice, though; or at least any engines would have needed a working `Symbol` before implementing this.